frontend/awx/administration/execution-environments/ExecutionEnvironmentPage/ExecutionEnvironmentDetails.cy.tsx

```diff
@@ -0,0 +1,76 @@
+import { ExecutionEnvironment } from '../../../interfaces/ExecutionEnvironment';
+import { ExecutionEnvironmentDetailInner as ExecutionEnvironmentDetails } from './ExecutionEnvironmentDetails';
+import { AwxItemsResponse } from '../../../common/AwxItemsResponse';
+import { formatDateString } from '../../../../../framework/utils/dateTimeHelpers';
+
+describe('ExecutionEnvironmentDetails', () => {
+  it('Component renders and displays Execution Environment', () => {
+    cy.fixture('execution_environments.json')
+      .then((eeResponse: AwxItemsResponse<ExecutionEnvironment>) => {
+        const execution_env: ExecutionEnvironment = eeResponse?.results[0];
+        return execution_env;
+      })
+      .then((eeObject) => {
+        cy.mount(<ExecutionEnvironmentDetails execution_env={eeObject} />);
+      });
+  });
```

Comment:
please add a .should assertion to the above test. 
.Contains by itself is not an assertion